src/librbd/operation/TrimRequest.cc

```diff
@@ -15,7 +15,7 @@
 #include "common/errno.h"
 #include "osdc/Striper.h"
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
Nit: just drop the include since it appears not to be used anymore

Reply:
Fixed